packages/itwin/measure-tools/src/tools/MeasureAreaTool.ts

```diff
@@ -110,6 +127,30 @@ MeasureAreaToolModel
     return EventHandled.Yes;
   }
 
+  private async sheetMeasurementsDataButtonDown(ev: BeButtonEvent, initial: boolean) {
```

Comment:
Not sure why the  'first' or 'initial' is needed at all. This needs to be done once. Function can be called directly after the `setMeasurementViewport` which also needs to be done once.